components/Container.tsx

```diff
@@ -7,26 +7,41 @@ interface ContainerProps {
   title?: string;
   className?: string;
   children: React.ReactNode;
+  noPadding?: boolean;
+  titleAbove?: boolean;
 }
 
-export const Container = ({ title, className, children }: ContainerProps) => {
+export const Container = ({
+  title,
+  className,
+  children,
+  noPadding,
+  titleAbove,
```

Comment:
We can set a default value for this
```suggestion
  titleAbove = false,
```